### PR TITLE
Add phishing incident edit layout

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Layouts/layout-edit-Phishing.json [hunk 1]
```diff
@@ -0,0 +1,293 @@
+{
+	"typeId": "Phishing",
+	"kind": "edit",
+	"layout": {
+		"id": "Phishing",
+		"version": 2,
```
Comment (aviadl) [inline Layouts/layout-edit-Phishing.json]: version should be -1
and also modified

File: Layouts/layout-edit-Phishing.json [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+		"modified": "2018-04-11T11:08:17.437020242Z",
+		"name": "",
+		"sections": [
+			{
+				"id": "",
+				"version": 0,
```
Comment (aviadl) [inline Layouts/layout-edit-Phishing.json]: Version should be also removed from here (or set to -1)

File: Layouts/layout-edit-Phishing.json [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+				"modified": "0001-01-01T00:00:00Z",
+				"name": "Basic Information",
+				"type": "",
+				"isVisible": true,
+				"readOnly": false,
+				"fields": [
+					{
+						"id": "",
+						"version": 0,
```
Comment (aviadl) [inline Layouts/layout-edit-Phishing.json]: And on these as well :)

File: Layouts/layout-edit-Phishing.json [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+						"isVisible": true
+					},
+					{
+						"id": "",
+						"version": 0,
+						"modified": "0001-01-01T00:00:00Z",
+						"fieldId": "incident_playbookid",
+						"isVisible": true
+					},
+					{
```
Comment (aviadl) [inline Layouts/layout-edit-Phishing.json]: Dont think we need labels on the edit here ...

File: Layouts/layout-edit-Phishing.json [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+						"id": "",
+						"version": 0,
+						"modified": "0001-01-01T00:00:00Z",
+						"fieldId": "incident_details",
+						"isVisible": true
+					},
+					{
```
Comment (aviadl) [inline Layouts/layout-edit-Phishing.json]: Also no need for attachment, since we got it from an external source, it is what it is

**Comment by meirwah** [comment]
@aviadl fixed comments, plz review again

**Pull request merged**